jina/peapods/pods/__init__.py

```diff
@@ -119,6 +121,38 @@ def close(self):
         """
         self.__exit__(None, None, None)
 
+    @staticmethod
+    def _set_upload_files(args):
+        # sets args.upload_files at the pod level so that peas inherit from it.
+        # all peas work under one remote workspace, hence important to have upload_files set for all
+
+        def valid_path(path):
+            try:
+                complete_path(path)
+                return True
+            except FileNotFoundError:
+                return False
+
+        _upload_files = set()
+        for param in ['uses', 'uses_internal', 'uses_before', 'uses_after']:
```

Comment:
is `uses_internal` still valid? @JoanFM

Reply:
I have a PR where I invalidate this, but still not merged